evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -700,7 +693,7 @@ def test_component_graph_evaluation_plumbing(
     graph = {
         "transformer a": [TransformerA, "X", "y"],
         "transformer b": [TransformerB, "transformer a.x", "y"],
-        "transformer c": [TransformerC, "transformer a", "transformer b.x", "y"],
+        "transformer c": [TransformerC, "transformer a.x", "transformer b.x", "y"],
```

Comment:
Ok so before "transformer a" was being silently ignored? I'm curious why this change doesn't mean we also have to change the actual expected call args for `mock_transc`

Reply:
Ohh it's cause the output of transformer b already had the output of transformer a. I wonder if we should change the test so that the column names don't overlap? Duplicate column names would cause ww to error out anyways.

Reply:
Hehe so right now, `parent_x = output_cache.get(parent_input, output_cache.get(f"{parent_input}.x"))` means that it'll try either getting `transformer a` or `transformer a.x` as a backup, which is why this worked. We're no longer going to have to default to this anymore but but https://github.com/alteryx/evalml/pull/2583 is going to clean that up further :)